### ISSUE TITLE
[Feature request] add distinct support

### ISSUE THREAD

**Issue by dmarcelino** [balderdashy/waterline#834]
Hi,

I've seen a `.distinct()` method being requested before (#60 and #122) but I can't seem to find it on the docs or code. Is there any support for it? Otherwise can it be added please?

Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @dmarcelino.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
